sdk/javascript/test/nem/MessageEncoder_spec.js

```diff
@@ -0,0 +1,55 @@
+const { PrivateKey, PublicKey } = require('../../src/CryptoTypes');
+const { KeyPair } = require('../../src/nem/KeyPair');
+const { MessageEncoder } = require('../../src/nem/MessageEncoder');
+const { MessageType, Message } = require('../../src/nem/models');
+const { runBasicMessageEncoderTests } = require('../test/messageEncoderTests');
+const { expect } = require('chai');
+
+describe('MessageEncoder (NEM)', () => {
+	runBasicMessageEncoderTests({ KeyPair, MessageEncoder, encodeAccessor: encoder => encoder.encode.bind(encoder) });
+
+	runBasicMessageEncoderTests({ KeyPair, MessageEncoder, encodeAccessor: encoder => encoder.encodeDeprecated.bind(encoder) });
+
+	it('decode falls back to input when cbc has wrong padding', () => {
+		// Arrange:
+		const keyPair = new KeyPair(PrivateKey.random());
+		const recipientPublicKey = new KeyPair(PrivateKey.random()).publicKey;
+		const encoder = new MessageEncoder(keyPair);
+		const encoded = encoder.encodeDeprecated(recipientPublicKey, (new TextEncoder()).encode('hello world'));
+
+		encoded.message[encoded.message.length - 1] ^= 0xFF;
+
+		// Act:
+		const [result, decoded] = encoder.tryDecode(recipientPublicKey, encoded);
+
+		// Assert:
+		/* eslint-disable no-unused-expressions */
```

Comment:
alternatively, use to.equal(false)

Reply:
I like `to.be.xxx` more, but changed given eslint complains and can get rid of those disable lines